internal/service/project/resource_project.go

```diff
@@ -411,9 +420,14 @@ func (r *projectRS) Create(ctx context.Context, req resource.CreateRequest, resp
 		resp.Diagnostics.AddError("error when getting project after create", fmt.Sprintf(ErrorProjectRead, projectID, err.Error()))
 		return
 	}
+	err = SetSlowOperationThresholding(ctx, connV2.PerformanceAdvisorApi, projectID, projectPlan.IsSlowOperationThresholdingEnabled)
+	if err != nil {
+		resp.Diagnostics.AddError(fmt.Sprintf("error when setting slow operation thresholding after create (%s): %s", projectID, err.Error()), "")
+		return
+	}
 
 	// get project props
-	projectProps, err := GetProjectPropsFromAPI(ctx, connV2.ProjectsApi, connV2.TeamsApi, projectID)
+	projectProps, err := GetProjectPropsFromAPI(ctx, connV2.ProjectsApi, connV2.TeamsApi, connV2.PerformanceAdvisorApi, projectID)
```

Comment:
another different endpoint consumed in project resource 😢 I understand a new resource for this can be overkill, but would be interesting to understand why the property was not added in https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Projects/operation/updateProject.

Reply:
as it's computed, in this case it would be only a data source